app/src/main/java/com/pajato/android/gamechat/chat/fragment/CreateGroupFragment.java

```diff
@@ -116,6 +119,14 @@
         // Post a welcome message to the default room from the owner.
         String text = "Welcome to my new group!";
         MessageManager.instance.createMessage(text, STANDARD, account, room);
+
+        // Dismiss the Keyboard and return to the previous fragment.
+        Activity a = getActivity();
```

Comment:
While I love the use of short variable names in Go, the accepted practice in Java is to use names a bit more revealing, fwiw.